src/Appwrite/Event/Mail.php

```
@@ -93,6 +94,28 @@ public function getBody(): string
+    {
+        $this->preview = $preview;
+
+        return $this;
+    }
+
+    /**
+     * Returns preview for the mail event.
+     *
+     * @return string
+     */
+    public function getPreview(string $preview): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$preview'. (Unused Code Rules)

(UnusedFormalParameter)